statsmodels/genmod/tests/results/results_glm.py

```diff
@@ -2210,7 +2221,7 @@ def __init__(self):
         super(InvGaussLog, self).__init__()
         filename = os.path.join(os.path.dirname(os.path.abspath(__file__)),
             "medparlogresids.csv")
```

Comment:
Many other files define once at the top of the file `cur_dir = os.path.dirname(os.path.abspath(__file__))`.  That along with cleaning up the indentation for these would be nice.  (and in a perfect world renaming "filename" --> "path")